Client/game_sa/CBuildingsPoolSA.cpp

```diff
@@ -210,15 +212,15 @@ bool CBuildingsPoolSA::Resize(int size)
     CBuildingSAInterface* newObjects = MemSA::malloc_struct<CBuildingSAInterface>(size);
     if (newObjects == nullptr)
```

Comment:
`if(!newObjects)`

Reply:
This is not mentioned in the **Style Guide**. And depends on the reviewer's personal preferences. BTW, in UE codebase `== nullptr` is preferable.

Reply:
for pointers `!` should be preferred as it takes less space and everyone knows what does it do. The same with `!strcmp`, but whatever

Reply:
This is discussable. And I didn't want to joint it. I just pointed out that there are many codestyles with theirs advantages and disadvantages. And since this particular code part isn't referenced in the guidelines, there is no reason to push too much.

Reply:
So, what do we keep here? I wrote this line automatically because all my commonly used languages require explicit null checks. Both variants are equal for me

Reply:
Well, its up to you until a new coding guidelines appear